OracleWebLogic/samples/12212-domain/container-scripts/createServer.sh

```diff
@@ -2,11 +2,24 @@
 #
 # Copyright (c) 2014-2015 Oracle and/or its affiliates. All rights reserved.
 #
+export DOMAIN_HOME=/u01/oracle/user_projects/domains/$DOMAIN_NAME
+echo "Domain Home: " $DOMAIN_HOME
+
+if [ -z $ADMIN_PASSWORD ]; then
+   echo "      ----> NO 'weblogic' admin password set"
+   echo ""
+else
+   s=${ADMIN_PASSWORD}
+   echo "      ----> 'weblogic' admin password: $s"
+fi
+sed -i -e "s|ADMIN_PASSWORD|$s|g" /u01/oracle/commonfuncs.py
```

Comment:
@Djelibeybi is there any security concern having the generated password set as environment variable? I mean... that information is accessible to any user in the host that can run `docker`, sure, but if there is such user, that user can also shutdown the container and do way worse things anyways... It's like trying to protect the kitchen from a thief who already got inside the house, and has plenty of time to inspect and workaround whatever is _securing_ the kitchen... :P